mentoring/answer.py

```diff
@@ -24,39 +21,146 @@
 # Imports ###########################################################
 
 import logging
-
 from lazy import lazy
 
-from xblock.fragment import Fragment
-
-from .light_children import LightChild, Boolean, Scope, String
 from .models import Answer
-from .utils import render_template
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Float, Integer, String
+from xblock.fragment import Fragment
+from xblock.validation import ValidationMessage
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from .step import StepMixin
+import uuid
 
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
 
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
```

Comment:
My understanding is that all the `Field` attributes such as the descriptions [are translated just prior to HTML rendering by the code that builds the settings editor UI](https://github.com/edx/edx-platform/blob/11dcf12f/common/lib/xmodule/xmodule/x_module.py#L1014). So XBlocks themselves don't need to translate those strings - they just need to mark them for translation using this method. Let me know if I'm missing something!